tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/Tests/GeneratorTests/sandwich/sandwich-BreadSetBreadEntity.dialog

```diff
@@ -12,7 +12,7 @@
         {
             "$kind": "Microsoft.SetProperty",
             "property": "$Bread",
-            "value": "@BreadEntity"
+            "value": "=@BreadEntity"
```

Comment:
>=@BreadEntity [](start = 22, length = 13)

Once this is checked-in, I'll need to update the generator.

Reply:
yes you will :)

---
In reply to: [367284059](https://github.com/microsoft/botbuilder-dotnet/pull/3253#discussion_r367284059) [](ancestors = 367284059)